src/datasets/iterable_dataset.py

```diff
@@ -941,7 +974,8 @@ def __init__(
         self.with_indices = with_indices
         self.input_columns = input_columns
         self.fn_kwargs = fn_kwargs or {}
-        self.formatting = formatting
+        self.formatting = formatting  # required for iter_arrow
+        self._return_features = return_features
```

Comment:
btw you can call this `features` (to be aligned with .map() for which `features=` corresponds to the output features)

Reply:
updated